docs/src/doc/dev/adr/0005-use-typescript.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Documentation|dev/ADRs/5) Use typescript" />
+
+# 5) Use TypeScript
+
+Date: 2020-07-16
+
+## Status
+
+Accepted
+
+## Context
+The components library that implements the design system will be implemented in Vue (see ADR #1). In addition to that we disucssied what benefits we can gain by using [TypeScript](https://www.typescriptlang.org/).
+
+There are several products (as of date) in the WMDE realm that were build with Vue, TypeScript and [Decorators](https://github.com/vuejs/vue-class-component): Termbox, Tainted References, Data Bridge, and we anticipate there will be more. This means the components library needs to support TypeScript apps.
```

Comment:
I think we should refer to what's called "Decorators" here as "Class Components" or "Class Component syntax". Decorators are only part of the package, and I'd only use the term when referring to the actual decorators like `@Component`, `@Prop` etc.

Reply:
Looks like that made it in. +1